flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/CatalogManager.java

```diff
@@ -1439,6 +1440,29 @@ public Set<String> listModels(String catalogName, String databaseName) {
         }
     }
 
+    /**
+     * Returns an array of names of temporary models registered in the namespace of the current
```

Comment:
init: it returns a set not an array. should it be a sorted array of names like some of the other list methods.